zhaquirks/tuya/builder/__init__.py

```diff
@@ -164,6 +174,22 @@ def tuya_battery(
         self.adds(power_cfg)
         return self
 
+    def tuya_illuminance(
+        self,
+        dp_id: int,
+        illuminance_cfg: TuyaLocalCluster = TuyaIlluminance,
+        converter: Optional[Callable[[Any], Any]] = None,
```

Comment:
This kwarg is never used, the default should be here, not in `.tuya_dp` unless you are certain that all Tuya devices require the scaling. If so, omit the kwarg.